### PR TITLE
Bootstrap French translation

### PR DESCRIPTION
**Pull request by fishilico**
Introduce French message file.

Closes: https://github.com/Polytechnique-org/xorgauth/issues/24

### CODE REVIEW

File: xorgauth/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+
+#: xorgauth/accounts/models.py:60
+#: xorgauth/accounts/oidc_provider_settings.py:35
+#: xorgauth/relying_party_test/templates/relying-party-test.html:39
+msgid "AX ID"
+msgstr "ID AX"
+
+#: xorgauth/accounts/models.py:60
+msgid "Identification in AX directory"
+msgstr "Identifiant and l'annuaire de l'AX"
```
Comment (vberger) [inline xorgauth/locale/fr/LC_MESSAGES/django.po]: s/and/dans/

File: xorgauth/locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+msgid "study year"
+msgstr "promotion"
+
+#: xorgauth/accounts/models.py:62
+msgid ""
+"Kind and main year of the study ('X1829' means 'entered the school in 1829 "
+"but 'M2005' means 'graduated in 2005')"
+msgstr ""
+"Catégorie et année de référence de la scolarité ('X1829' signifie 'est "
+"entréà l'école en 1829 mais 'M2005' signifié 'diplomé en 2005')"
```
Comment (vberger) [inline xorgauth/locale/fr/LC_MESSAGES/django.po]: s/entréà/entré à/

**Comment by vberger** [comment]
Bon pour moi modulo les deux typos 👍

**Comment by fishilico** [comment]
Thanks for your review. I fixed the typos.

**Pull request merged**